smoderp2d/.config.ini

```diff
@@ -22,6 +22,8 @@ printtimes:
 level: DEBUG
 
 [processes]
-# Mfda enabled
```

Comment:
@pesekon2 MFDA can be remove from hidden config, right?

Reply:
@landam: Yes, it can and should be removed. It is probably my fault it's still there.